pyproject.toml

```diff
@@ -42,6 +42,7 @@ python = "^3.7.1"
 sphinx-panels = {version = "^0.5.2", optional = true}
 sphinx-rtd-theme = {version = "^0.5.1", optional = true}
 tqdm = "^4.59.0"
+tensorflow-cpu = {version = "^2.4.1", optional = true}
 
```

Comment:
Need a manual entry in `pyproject.toml` in this case (i guess there is a way to do it from CLI and i am oblivious to that). Need to create a `tesnorflow` extra inside `[tool.poetry.extras]` (refer line 55) so that user can do:
```
pip install optical[tensorflow]
```

Reply:
Made changes

Reply:
You added `tensorflow-cpu` to `docs` extra :). Please add another entry for `tensorflow` extra and add `tensorflow-cpu` to that list

Reply:
@bishwarup307 , it seems like poetry way of add extra dependency is `poetry add  tensorflow-cpy -E tensorflow`, the result will be `tensorflow-cpu = {version = "^2.4.1", extras = ["tensorflow"]}`. But still we should manually edit the `[tool.poetry.extras]` section.